cookbooks/aws-parallelcluster-config/recipes/aws_cli.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+#
+# Cookbook:: aws-parallelcluster
+# Recipe:: aws_cli
+#
+# Copyright:: 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance with the
+# License. A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+# OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Users to configure the AWS CLI for.
+# We configure the AWS CLI for all the users that are created by ParallelCluster and must interact with AWS.
+users = node["cluster"]["head_node_imds_allowed_users"]
```

Comment:
I think we are using single quotes to wrap variable names. I don't get why the linter doesn't notify about this

Reply:
Got it, fixing